model.py

```diff
@@ -29,13 +30,29 @@ class Shortlist:
 
 
 #functions
+pickle_file_name = "shortlist.pickle"
+
+def load_pickle(path):
+    with open(path/pickle_file_name, "rb") as f:
+        shortlist = pickle.load(f)
+    return shortlist
+
+def save_pickle(path,shortlist):
```

Comment:
`save_shortlist` would be a better name, mirroring `load_shortlist`. It also hides inside `model.py` _how_ the shortlist is saved.